serf/serf.go

```diff
@@ -25,7 +25,7 @@ import (
 // version to memberlist below.
 const (
 	ProtocolVersionMin uint8 = 2
-	ProtocolVersionMax       = 4
+	ProtocolVersionMax       = 5
```

Comment:
Can you elaborate on this? Nothing here should be backwards-incompatible, so it doesn't seem like this should affect Consul upgrades at all.